llvm/test/tools/llvm-objcopy/ELF/gap-fill-order.test

```diff
@@ -0,0 +1,34 @@
+# RUN: yaml2obj %s >%t
+
+# Test gap fill with all allocatable output sections
```

Comment:
Nit: New llvm-objcopy tests use `##` for the comments, to help them stand out from RUN and CHECK lines. 

Also, this sentence needs a trailing "." - this applies in several other places in the tests too.

This comment doesn't really help me understand the purpose of this test. The test name is `gap-fill-order.test` which implies that it has something to do with the order of something, but it's not clear what that is.

Reply:
Fixed the comments and some punctuation in comments and added a comment.